.travis.yml

```diff
@@ -4,12 +4,18 @@ python:
 before_install:
```

Comment:
Also, travis complaining about being unable to parse travis.yml. You may want to check that out. Without that, travis checks are not triggered at all.

Reply:
I have fixed it now